chart/env/prod.yaml

```diff
@@ -537,3 +541,31 @@ workers:
       limits:
         cpu: 2
         memory: "1Gi"
+
+webhook:
+  # Number of uvicorn workers for running the application
+  # (2 x $num_cores) + 1
+  # https://docs.gunicorn.org/en/stable/design.html#how-many-workers
+  uvicornNumWorkers: "9"
+  nodeSelector:
+    role-datasets-server-webhook: "true"
+  tolerations:
+    - key: "huggingface.co/datasets-server-webhook"
+      operator: "Exists"
+      effect: "NoSchedule"
+  replicas: 20
```

Comment:
I don't think we need so many of them

Reply:
I put 4 replicas, but I'm not sure if that is enough, can you advise?

Reply:
let's monitor